Wrappers/Python/ccpi/viewer/CILViewer.py

```diff
@@ -1148,7 +1146,24 @@ def setInterpolateOff(self):
         self._viewer.imageSlice.GetProperty().SetInterpolationTypeToNearest()
         self.renWin.Render()
 
-    def setColourWindowLevel(self, window, level):
+    def setSliceColorWindowLevelPercentiles(self, min_percentage, max_percentage):
+        min_val, max_val = self.getVolumeMapWindow((min_percentage, max_percentage), 'scalar')
```

Comment:
I think I commented on `getVolumeMapWindow` already with @lauramurgatroyd . I think it hides inside the `vtkHistogramStatistics` which calculates min and max from the percentiles. Can't we create a method to return the min and max?

Reply:
Is that not what the getVolumeMapwindow function does?

Reply:
At least that is how I have been using it by passing min_percentage=0 and max_percentage=100

Reply:
@paskino - do you mean this?
https://github.com/vais-ral/CILViewer/blob/6ac6efa69bd1e15ce0ef8cbca77fe6e5c572fdf5/Wrappers/Python/ccpi/viewer/CILViewer.py#L1025-L1034

Reply:
automatically gets the full range

Reply:
The reason it has volume is because it is specifying that it is not just for the slice.